routes/data.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+        "title": "Frequently asked Interview Questions in react-redux",
+        "href": "https://blog.zairza.in/frequently-asked-interview-questions-in-react-redux-bc774733b449?source=collection_home---5------0-----------------------",
+        "author": "Jayashree Panda",
+        "release": "2019-11-22T08:11:26.615Z"
+    },
+    {
+        "title": "OAuth using MEVN Stack",
+        "href": "https://blog.zairza.in/oauth-using-mevn-stack-4b4a383dae08?source=collection_home---6------0-----------------------",
+        "author": "Ramakrishna Pattnaik",
+        "release": "2019-08-25T12:13:49.122Z",
+        "cover": "https://cdn-images-1.medium.com/fit/t/1600/480/1*zqCh8ZNR-LjBzaacpiIyUA.png"
```

Comment:
Since you are fetching the cover image, it's huge. Which is why it's zoomed out. We need the image which the first image inside the blog. WDYT? Can this be done?

Reply:
Yes, I think so. We need to scrape every hrefs of particular blog to get the right image. Working on it now.

Reply:
Mostly we'll do this step for 4 of the recent posts for optimization.

Reply:
yeah, that should reduce unnecessary requests